web/src/beta/lib/core/StoryPanel/Block/builtin/Timeline/hook.ts

```diff
@@ -101,6 +110,30 @@ export default ({
     return {};
   }, [range]);
 
+  const panelSettings = useMemo(() => {
+    if (!property?.panel) return undefined;
+    return {
+      padding: {
+        ...property?.panel?.padding,
+        value: calculatePaddingValue(DEFAULT_BLOCK_PADDING, property?.panel?.padding?.value),
+      },
+    };
+  }, [property?.panel]);
+
+  const paddingCheck = useMemo(() => {
+    return Math.max(
+      padding?.left || 0,
+      padding?.right || 0,
+      panelSettings?.padding.value.left || 0,
+      panelSettings?.padding.value.right || 0,
+    );
+  }, [
+    padding?.left,
+    padding?.right,
+    panelSettings?.padding.value.left,
+    panelSettings?.padding.value.right,
+  ]);
```

Comment:
What I undestand is happening here is that you are cross-referencing the storypanel's and the block's left/right padding and using whichever is highest for the block's padding.
Is that correct? If its correct, that isn't how it should be, right? Storypanel should use the storypanel's padding, and each block should use their own padding...
Maybe I'm misunderstanding something 🤔

Reply:
@KaWaite Yes you are right but both of padding (storypanel's padding, and  block padding) are needed. for example if padding is applied in a block I use `panelSettings?.padding.value.left || 0, panelSettings?.padding.value.right || 0,` to check for the specific block padding. and the remaining timeline block are not affected by this padding value